scripts/artifacts-building/apt/apt-artifacts-testing.yml

```diff
@@ -0,0 +1,60 @@
+---
+- hosts: localhost
+  connection: local
+  gather_facts: no
+  vars:
+    distro: "trusty"
+    rpc_version: "r14.0.0rc1"
```

Comment:
It should be ``rpc_release`` to match the var in the user_* files. That said, it might be better to not define a default for this to ensure that it fails if a value is not provided.